src/liboslexec/llvm_util.cpp

```diff
@@ -570,9 +696,11 @@ LLVM_Util::setup_optimization_passes (int optlevel)
         mpm.add (llvm::createVerifierPass());
         // Simplify the call graph if possible (deleting unreachable blocks, etc.)
         mpm.add (llvm::createCFGSimplificationPass());
+#if OSL_LLVM_VERSION <= 36
```

Comment:
`createScalarReplAggregatesPass` doesn't exist anymore, or if it does I cannot find it.

Updated with a comment and moved the **#ifdef** a line lower (where it should have been to begin with).